src/mount/oplog.cc

```diff
@@ -76,23 +75,23 @@ static inline void oplog_put(uint8_t *buff, uint32_t leng) {
 	}
 #endif
 	uint32_t bpos;
-	if (leng > OPBUFFSIZE) { // just in case
-		buff += leng - OPBUFFSIZE;
-		leng = OPBUFFSIZE;
+	if (leng>OPBUFFSIZE) {  // just in case
```

Comment:
Do you have .clang-format file correctly configured in your environment in order to keep proper spacing around operators, function arguments etc. for code readability? Consider to check the main changes made on oplog.cc file to check this code format issue.

Reply:
If the line changes are not relevant for what the commit is attempting to achieve, they should not be changed because this pollutes the blame history with non-relevant changes that will be difficult differentiate. This reverses a previous commit changes (as I requested for the previous reason) https://github.com/leil-io/saunafs/pull/24#discussion_r1521748912 

Hopefully before the next release we could do the style changes (even if they are disruptive) and filter them out in [.git-blame-ignore-revs](https://github.com/leil-io/saunafs/blob/main/.git-blame-ignore-revs).